src/components/Modal/ModalConfirmation/ModalConfirmation.tsx

```diff
@@ -8,7 +8,7 @@ import { IconXClose } from '../../Icons';
 
 import styles from './ModalConfirmation.module.scss';
 
-export interface ModalConfirmationProps {
+export interface ModalConfirmationProps extends DialogProps {
```

Comment:
Was this preventing modal closing `onCancel` (click cross icon) ?

Reply:
no, it's working good.

I want to use the `open`  prop from `DialogProps` which it was not in `ModalConfirmationProps`